src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/publicApi/model/licence/additionalConditions/MultipleExclusionZoneAdditionalCondition.kt

```diff
@@ -0,0 +1,38 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.publicApi.model.licence.additionalConditions
```

Comment:
<kbd>
<img width="1028" alt="Screenshot 2023-10-27 at 11 47 39" src="https://github.com/ministryofjustice/create-and-vary-a-licence-api/assets/1517745/8f386fcf-b3e6-4374-a7e5-c2e555c076ef">
</kbd>